Python/formatter_unicode.c

```diff
@@ -280,7 +281,10 @@ parse_internal_render_format_spec(PyObject *format_spec,
 
     if (end-pos > 1) {
         /* More than one char remain, invalid format specifier. */
-        PyErr_Format(PyExc_ValueError, "Invalid format specifier");
+        PyErr_Format(PyExc_ValueError,
+                     "Invalid format specifier: '%s' for object of type '%s'",
+                     PyUnicode_AsUTF8AndSize(format_spec, NULL),
```

Comment:
Not that it makes much difference, but you can use PyUnicode_AsUTF8 here.

I'm not sure PyError is expecting UTF-8 (and not ascii). I'll have to research and see what other error messages do.

Reply:
Replaced with `PyUnicode_AsUTF8`.